.github/CODEOWNERS

```diff
@@ -0,0 +1,32 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+
+# Prefer GitHub teams over individuals to make it easier to find a suitable owner
+# when someone is OOO or leaves the team.
+
+# terraform-samples-owners is in charge of infrastructure in this repository ##### TODO: create this GitHub team #####
+# terraform-samples-reviewers reviews Terraform sample code for adherence to sample guidelines
+*                                      @GoogleCloudPlatform/terraform-samples-owners @GoogleCloudPlatform/terraform-samples-reviewers
```

Comment:
I don't know if we can add teams from other orgs to access repos in this org. Keep this PR in draft until we verify, or until we create the corresponding terraform-google-modules teams